doc/design/interceptors.md

```diff
@@ -9,60 +16,136 @@ Interceptors are components of the Flume agent architecture. Typically, such an
+                ],
+                "regex": "Room\.(\D*)Room",
+                "destination": "character_rooms",
+                "fiware_service_path": "rooms"
+            },
+            {
+                "id": 3,
+                "fields": [
+                    "entityType",
+                    "entityId"
+                ],
+                "regex": "RoomRoom\.(\D*)",
+                "destination": "character_rooms",
+                "fiware_service_path": "rooms"
+            },
+            {
+                "id": 4,
+                "fields": [
+                    "entityType"
+                ],
+                "regex": "Room",
+                "destination": "other_rooms",
+                "fiware_service_path": "rooms"
+            }
+        ]
+    }
 
 The above rules set that:
 
-* All the `Room` entities having their identifiers composed by a `Room.` and an integer (e.g. `Room.12`) will be persisted in a `numeric_rooms` destination entity within a `rooms` destination dataset (in the example, the concatenation is equals to `Room.12Room`).
-* All the `Room` entities having their identifiers composed by a `Room.` and any number of characters (no digits) (e.g. `Room.left`) will be persisted in a `character_rooms` destination entity within a `rooms` destination dataset (in the example, the concatenation is equals to `Room.leftRoom` when appliying rule number 2, but `RoomRoom.left` when applying rule number 3; nevertheless, from a semantic point of view they are the same rule).
-* All other rooms will go to `other_rooms` within a `rooms` destination dataset.
+* All the `Room` entities having their identifiers composed by a `Room.` and an integer (e.g. `Room.12`) will be persisted in a `numeric_rooms` destination within a `rooms` service parth (in the example, the concatenation is equals to `Room.12Room`).
+* All the `Room` entities having their identifiers composed by a `Room.` and any number of characters (no digits) (e.g. `Room.left`) will be persisted in a `character_rooms` destination within a `rooms` service path (in the example, the concatenation is equals to `Room.leftRoom` when appliying rule number 2, but `RoomRoom.left` when applying rule number 3; nevertheless, from a semantic point of view they are the same rule).
+* All other rooms will go to `other_rooms` destination within a `rooms` service path.
+
+Rules are tryed sequentially, and if any rules matches then the default destination for the notified entity is generated, i.e. the concatenation of the entity id, `_` and the entity type; and the notified service path is maintained.
+
+[Top](#top)
 
-Rules are tryed sequentially, and if any rules matches then the default destination for the notified entity is generated, i.e. the concatenation of the entity id, `_` and the entity type.
+###<a name="section3.2"></a>Usage
 
 The matching table file is usually placed at `[FLUME_HOME_DIR]/conf/`, and there exists a template within Cygnus distribution.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/2d7a441a6c25cdcb43f78808264d710175523a8f